lib/VM/JSLib/RegExp.cpp

```diff
@@ -460,6 +461,65 @@ static void createGroupsObject(
   JSObject::setNamedSlotValueUnsafe(matchObj.get(), runtime, groupsDesc, shv);
 }
 
+static void createIndicesArray(
+    Runtime &runtime,
+    Handle<JSArray> matchObj,
+    Handle<JSObject> mappingObj,
+    Handle<JSArray> indices) {
+  // If there are no capture groups, then set groups to undefined.
+  if (!mappingObj) {
```

Comment:
small nit- is there a point to flipping this conditional if both cases are written out?

Reply:
It was just preference I guess, I can flip it if you like :)

Reply:
Again it's a nit, but I personally feel like it would be cleaner without the negation.